mods/ts/rules/structures.yaml

```diff
@@ -1256,8 +1238,7 @@ NAPULS:
 	Buildable:
 		Queue: Defense
 		BuildPaletteOrder: 90
-		Prerequisites: radar
-		Owner: nod, gdi
+		Prerequisites: radar, ~structures.gdi
```

Comment:
http://cnc.wikia.com/wiki/Electromagnetic_Pulse_cannon is used by every faction.